<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="F2lineEdit">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>50</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="F3label">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>9</y>
      <width>123</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency of third signal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="F3lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaxTimelineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MaxTimelabel">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>9</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="F2label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>9</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency of second signal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FreqDiscrLineEdit">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>50</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FreqDiscrlabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency of Discretization:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="F1label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>9</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency of first signal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="F1lineEdit">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>50</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetGraph1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>696</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetGraph2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>696</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetGraph3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>696</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetGraphSum" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>696</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetGraphDFT" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>696</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
